turcica.csl

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>turcica</title>
+    <id>http://www.zotero.org/styles/turcica</id>
+    <link rel="self" href="http://www.zotero.org/styles/turcica"/>
+	<link href="https://poj.peeters-leuven.be/content.php?journal_code=TURC&url=journal" rel="documentation"/>
+    <author>
+      <name>Lucas Equinoxe</name>
+      <email>lucas.equinoxe@college-de-france.fr</email>
+    </author>
+	<issn>0082-6847</issn>
+    <eissn>1783-1822</eissn>
+    <updated>2022-09-23T07:46:36+00:00</updated>
+  </info>
```

Comment:
Please add this line back after line 14:
`    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
`